app/mixins/ember-table-route.js

```diff
@@ -51,8 +51,6 @@ export default Mixin.create({
   applySortFilters(query, params) {
     if (params.sort_by && params.sort_dir) {
       query.sort = `${params.sort_dir === 'ASC' ? '-' : ''}${params.sort_by}`;
-    } else {
-      delete query.sort;
```

Comment:
Why?

Reply:
Thus line is deleting query.sort if params.sort_by and params.sort_dir are not present. So if I am not overriding those parameters they get deleted once applySortFilters method is called.

Reply:
But they will be present because you have added `params.sort_by || 'starts-at'`. Under no circumstance can that be undefined. This will break the site. We had this for a reason, we don't add things randomly and thus cannot just remove somethings without consequences

Reply:
No, query.sort will be present not params.sort_by and params.sort_dir. They will be undefined until we change that from website as you have said not to override params.
Other way could be to check for query.sort to be present in mixins/ember-table-route.js instead of checking for params.sort_by and parama.sort_dir.

Reply:
See how other tables have defined a default sorting order